libs/vscode-extension/src/lib/models/project.ts

```diff
@@ -53,6 +57,7 @@ export interface IProjectWizardContext extends IActionContext {
   languageFilter?: RegExp;
   workerRuntime?: IWorkerRuntime;
   openBehavior?: OpenBehavior;
+  workspaceName?: string;
```

Comment:
Cant we get the workflowspace name from workspace path?

Reply:
These have already been PRd, this is just merge their dev branch into main. Talk to @samikay101 about any changes you’d request.